internal/persistence/sql/persister.go

```diff
@@ -70,6 +70,15 @@ func (p *Persister) Connection(ctx context.Context) *pop.Connection {
 	return popx.GetConnection(ctx, p.conn.WithContext(ctx))
 }
 
+func (p *Persister) Ping() error {
+	type pinger interface {
+		Ping() error
+	}
+
+	// This can not be contextualized because of some gobuffalo/pop limitations.
+	return errors.WithStack(p.conn.Store.(pinger).Ping())
```

Comment:
This doesn't work because the "original" `Store` is wrapped in a `contextStore`, which for some reason doesn't implement Ping anymore: `interface conversion: pop.contextStore is not sql.pinger: missing method Ping`

In Kratos we don't do that. Not sure why we do it Keto.